### DISCUSSION TITLE
Diadora Fit Bike

### DISCUSSION THREAD

**Discussion by cagnulein** [doudar/SmartSpin2k#298] [General]
Hi @doudar, i will add here the measurement and photos about Diadora Fit Bike https://www.diadorafitness.it/fit-bike.html
If you think, we can move this on a new issue.

**Comment by cagnulein** [comment]
![IMG_0639](https://user-images.githubusercontent.com/48216/148086335-ed867ec2-e52a-4e0b-81af-13c29562ad07.jpg)
![IMG_0638](https://user-images.githubusercontent.com/48216/148086350-5334ead7-f75e-4f3a-896a-eabf03b5ae8b.jpg)
![IMG_0637](https://user-images.githubusercontent.com/48216/148086354-b942f7b8-f2e7-43c0-827a-db02b3e6c2e0.jpg)
![IMG_0640](https://user-images.githubusercontent.com/48216/148086360-647cef58-228c-4d6d-a2db-2d79e8573de2.jpg)

**Comment by cagnulein** [comment]
Here they are the measurement:
> 1. The distance from the center of the knob to the back of the head tube (Critical measurement because if it's less than 49mm it may not fit).

It's 49mm :( is it enough?

> 2. The largest diameter of the knob (Outside-Outside of the finger grooves) (a straight-on top down photo so I can see how many lobes are on the knob)

49mm

> 3. The smallest Diameter of the knob (inside-inside of the finger grooves).

44mm

let me know if you need other infos

**Reply by doudar** [reply]
Ouch! 49mm is the absolute minimum but I also see that the head tube rakes back as well which probably makes it an even tighter fit. 

Let me think about this a bit. Maybe I can modify the case design a little bit to see if I can shorten it up any more specifically for your bike.

**Comment by cagnulein** [comment]
Don't lose your day about this. If it's easy ok, otherwise I will put this
shit of bike in the garbage:D--
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Reply by doudar** [reply]
#299 created just for you and your junk bike 🤣

Just kidding of course. 😅

I actually wanted to try a couple of the things there anyway. Printing it right now and I’ll see if the components fit in it. If all works as planned, it will be ~10mm shorter.

**Comment by doudar** [comment]
@cagnulein 

I'm about to design the insert, and looking at the photos, it appears possibly you read the caliper incorrectly? 

In the pictures, I see 59mm head tube to knob center;
59mm outside to outside knob (largest diameter);
54mm inside to inside (smallest diameter);

Please confirm.

**Comment by cagnulein** [comment]
@doudar OMFG! i'm so sorry! i did it during the booster vaccine day and i guess i was out of my mind. You're absolutely right :( So the job that you did to small the case is useless!? i'm so so sorry :(
To be sure i will do the measurements this evening too, just to confirm this again

**Reply by doudar** [reply]
No problem! Like I said, I wanted to make a couple small design changes anyway :)

That's one of the reasons backing up the measurements with pictures is necessary .

**Reply by cagnulein** [reply]
@doudar confirmed: 59mm 59mm 54mm

**Comment by cagnulein** [comment]
![image](https://user-images.githubusercontent.com/48216/150598313-c14b464d-9a94-4cfd-a4de-328831bbec51.jpeg)

I can't ride yet because my wife is sleeping! :D

**Comment by doudar** [comment]
Does it fit okay? You’ll also need to secure the arm to your head tube with
either the Velcro I included or something else.

On Fri, Jan 21, 2022 at 2:51 PM Roberto Viola ***@***.***>
wrote:

> [image: image]
> <https://user-images.githubusercontent.com/48216/150598313-c14b464d-9a94-4cfd-a4de-328831bbec51.jpeg>
>
> I can't ride yet because my wife is sleeping! :D
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/doudar/SmartSpin2k/discussions/298#discussioncomment-2018746>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEEO2KF7LUJK5M7OD7YVBMLUXHBNPANCNFSM5LHXUETQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 

*----------------------*
*Anthony Doud*

**Comment by cagnulein** [comment]
Yes! It’s perfect! Yes in the photo I removed the Velcro because I was
playing with it :)
I already moved it mounted to the knob and it works flawlessly! Tomorrow I
will try my first auto resistance ride!
Thank you so much!

Il giorno ven 21 gen 2022 alle 21:58 Anthony Doud ***@***.***>
ha scritto:

> Does it fit okay? You’ll also need to secure the arm to your head tube with
> either the Velcro I included or something else.
>
> On Fri, Jan 21, 2022 at 2:51 PM Roberto Viola ***@***.***>
> wrote:
>
> > [image: image]
> > <
> https://user-images.githubusercontent.com/48216/150598313-c14b464d-9a94-4cfd-a4de-328831bbec51.jpeg
> >
> >
> > I can't ride yet because my wife is sleeping! :D
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/doudar/SmartSpin2k/discussions/298#discussioncomment-2018746
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AEEO2KF7LUJK5M7OD7YVBMLUXHBNPANCNFSM5LHXUETQ
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
> --
>
> *----------------------*
> *Anthony Doud*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/doudar/SmartSpin2k/discussions/298#discussioncomment-2018778>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWGGF7ZFSVNEOD22JS3UXHCFVANCNFSM5LHXUETQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud
